homeassistant/bootstrap.py

```diff
@@ -253,6 +248,7 @@ def from_config_dict(config: Dict[str, Any],
     if skip_pip:
         _LOGGER.warning('Skipping pip installation of required modules. '
                         'This may cause issues.')
+        pkg_util.mount_lib_path(hass.config.config_dir)
```

Comment:
Not sure why we would move it here? This should only be done if we set the config dir.